quaddtype/pyproject.toml

```diff
@@ -12,8 +12,19 @@ name = "numpy_quaddtype"
 description = "Quad (128-bit) float dtype for numpy"
 version = "0.2.0"
 readme = 'README.md'
-license = { file = "LICENSE" }
+license = "BSD-3-Clause"
+license-files = ["LICENSE"]
 authors = [{name = "Swayam Singh", email = "singhswayam008@gmail.com"}]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3.14",
```

Comment:
We are also compatible with free-threading python so might also can add `Programming Language :: Python :: Free Threading`